src/factories/calorimetry/HEXPLIT_factory.h

```diff
@@ -20,6 +20,7 @@ class HEXPLIT_factory : public JOmniFactory<HEXPLIT_factory, HEXPLITConfig> {
 
     ParameterRef<double> m_MIP      {this, "MIP",           config().MIP};
     ParameterRef<double> m_Emin_in_MIPs     {this, "Emin_in_MIPs",          config().Emin_in_MIPs};
+    ParameterRef<double> m_delta_in_MIPs     {this, "delta_in_MIPs",          config().delta_in_MIPs};
     ParameterRef<double> m_tmax     {this, "tmax",          config().tmax};
```

Comment:
Formatting
```suggestion
    ParameterRef<double> m_MIP {this, "MIP", config().MIP};
    ParameterRef<double> m_Emin_in_MIPs {this, "Emin_in_MIPs", config().Emin_in_MIPs};
    ParameterRef<double> m_delta_in_MIPs {this, "delta_in_MIPs", config().delta_in_MIPs};
    ParameterRef<double> m_tmax {this, "tmax", config().tmax};
```

(I know I'm being tedious, but small things like formatting really help us in the long run!)

Reply:
done, thanks